### ISSUE TITLE
Map jumping off the div when made responsive

### ISSUE THREAD

**Issue by Vishal0203** [markmarkoh/datamaps#233]
As the heading says, I followed the documentation and added the map.resize() method. But, the map doesn't stay in the grid since then. Is that a bug?

**Comment by hoihoi661** [comment]
Somewhat had the same problem, just added .datamap{left: 0;} to my CSS which surprisingly seemed to have solved all my problems (far as I know).

**Comment by vishal-kony** [comment]
Yeah, looks like the chart starts from the extreme left of the div. I had `<center>` tag in my div and that was causing the map jumping off.
